### PR TITLE
[#13230] Show default syncing interval

### LINKED ISSUE #13230: Selected default value of Sync history period is not shown in Sync settings (Android)
Selected default value of Sync history period is not displayed in Sync settings. 

### Reproduction

1) Open Status
2) Login an account with default unchanged Sync history period value
3) Go to Profile - Sync settings - Sync history for
4) Pay attention if any value is selected

#### Actual behavior
Non of values is displayed as selected. In fact, the default "One day" value is selected.  Reproducible only for Android. 

https://user-images.githubusercontent.com/97245802/160649602-380e52a3-c0dc-422f-92e8-b069ac6fc945.mp4

#### Expected behavior
Selected value is shown

### Additional Information

- Status version: nightly 
- Operating System: Android

### PR DESCRIPTION
**Pull request by rasom**
fix #13230

status: ready

### CODE REVIEW

**Review by Parveshdhull — APPROVED** [review]

File: src/status_im/ui/screens/default_sync_period_settings/view.cljs [hunk 1]
```diff
@@ -25,7 +25,7 @@
     [react/view {:margin-top 8}
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/default_sync_period_settings/view.cljs:25]: `(views/letsubs [{:keys [default-sync-period] :or {default-sync-period constants/one-day}} [:multiaccount]]`
will be more clean, I guess

File: src/status_im/ui/screens/default_sync_period_settings/view.cljs [hunk 2]
```diff
@@ -25,7 +25,7 @@
...
      (when config/two-minutes-syncing?
        [radio-item constants/two-mins default-sync-period])
-     [radio-item constants/one-day default-sync-period]
+     [radio-item constants/one-day (or default-sync-period constants/one-day)]
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/default_sync_period_settings/view.cljs]: `(views/letsubs [{:keys [default-sync-period] :or {default-sync-period constants/one-day}} [:multiaccount]]`
will be more clean, I guess

Comment (rasom) [reply src/status_im/ui/screens/default_sync_period_settings/view.cljs]: I wrote that macro and I'm still afraid of touching it when it is not 100% necessary :D let me try

Comment (rasom) [reply src/status_im/ui/screens/default_sync_period_settings/view.cljs]: done

**Comment by pavloburykh** [comment]
@rasom thanx for PR. Ready for merge.

**Pull request merged**